Sourabh_Kanwade/html/Assignment 6/Question_2.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Form - Professional Details</title>
+        <link rel="stylesheet" href="./question_2.css" />
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <hr />
+        <h2>Personal Details</h2>
+        <label for="first-name">First Name: </label>
+        <input type="text" name="fname" id="first-name" placeholder="First Name" />
```

Comment:
placeholders second words 1st letter should be small like First name make this change in all input boxes